packages/strapi-design-system/src/Alert/Alert.js

```diff
@@ -140,5 +140,5 @@ Alert.propTypes = {
 };
 
 AlertIcon.propTypes = {
-  variant: Alert.propTypes.variant,
+  variant: Alert.propTypes.variant.isRequired,
```

Comment:
shouldn't we leave the default variant defined by `defaultProps`? 👀

Reply:
We don't export `AlertIcon` and the default is defined on it's parent, so i think it's probably fine to leave as is.